src/mango_httpd.erl

```diff
@@ -157,6 +165,15 @@ get_idx_create_opts(Opts) ->
     end.
 
 
+get_bulk_delete_ddocs(Opts) ->
```

Comment:
You might rename this to get_bulk_delete_ddoc_ids. It was a bit confusing on the first read through.